editors/code/README.md

```diff
@@ -30,4 +48,26 @@ Features:
 
 [manual]: https://rust-analyzer.github.io/manual.html
 
-See https://rust-analyzer.github.io/ for more information.
+## Quick start
+
+1. Install [rustup] the rust toolchain
```

Comment:
```suggestion
1. Install [rustup]
```

Reply:
Right, I copied that from the official rust extension, but just saying rustup is indeed nicer.

done